spec/support/workspace_helper.rb

```diff
@@ -23,22 +23,12 @@
   attr_reader :workspace_dir, :workspace_content_dir, :workspace_metadata_dir
 end
 
-def create_xml_file(filename)
-  copy_fixture_file('001.xml', filename)
-end
-
-def create_pdf_file(filename)
-  copy_fixture_file('001.pdf', filename)
-end
-
-def create_txt_file(filename)
-  copy_fixture_file('001.txt', filename)
-end
-
-def create_tif_file(filename)
-  copy_fixture_file('001.tif', filename)
+def create_ocr_file(filename)
+  fixture_file = "001#{File.extname(filename)}"
+  FileUtils.copy(Pathname('spec') / 'fixtures' / 'ocr' / fixture_file, Pathname(workspace_content_dir) / filename)
 end
 
-def copy_fixture_file(fixture_file, filename)
-  FileUtils.copy(Pathname('spec') / 'fixtures' / 'ocr' / fixture_file, Pathname(workspace_content_dir) / filename)
+def create_speech_to_text_file(filename)
+  fixture_file = "file1#{File.extname(filename)}"
+  FileUtils.copy(Pathname('spec') / 'fixtures' / 'speech_to_text' / fixture_file, Pathname(workspace_content_dir) / filename)
 end
```

Comment:
refactor these helpers methods to make them less copy/pasta

that also accounts for the changes in the spec above when calling these methods